packages/sakuli-legacy/package.json

```diff
@@ -55,7 +55,7 @@
     "@types/properties-reader": "0.0.1",
     "@types/selenium-webdriver": "4.0.0",
     "@types/yargs": "^12.0.12",
-    "chromedriver": "78.0.0",
+    "chromedriver": "^80.0.1",
```

Comment:
Check if chrome driver version is compatible with CI/CD environment.
Please ensure that the chrome driver version is consistent in the whole project.